composer.json

```diff
@@ -23,6 +23,9 @@
     "require": {
         "php": ">=5.2.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "4.*"
```

Comment:
yeah, the only reason I avoided 5 is because this lib is designed to run on <= 5.6, including 5.5, 5.4 and 5.3...